packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -1172,7 +1251,7 @@ void main() {
         ..rrect(color: const Color(0x0a000000))
         ..rrect(color: const Color(0xffffffff)),
     );
-  });
+  }, variant: TargetPlatformVariant.only(TargetPlatform.macOS));
```

Comment:
Why is this Mac only now?

Reply:
I had focus and focus tests only on macOS, I am not sure if that is good or bad (or what the consensus/best practice is).

Reply:
I have removed the macOS requirement for focus.